src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.ts

```diff
@@ -72,7 +72,7 @@ export class LocalGameWrapperComponent extends GameWrapper implements AfterViewI
         const gameStatus: GameStatus = this.gameComponent.rules.getGameStatus(this.gameComponent.rules.node);
         if (gameStatus.isEndGame === true) {
             this.endGame = true;
-            if (gameStatus.winner !== Player.NONE) {
+            if (Player.isPlayer(gameStatus.winner)) {
```

Comment:
Lu comme ça j'adhérerais plus à un "gameStatus.winner.isPresent()" mais du coup ça serait plutôt out-of-scope, quoique ?

Reply:
c'est-y-pas mieux avec gameStatus.winner.isPlayer() ?

Reply:
Mmh non, c'est moins clair si tu ignore qu'on veut dire "isInstanceOfPlayerBecauseNotPlayerNone" alors que dans l'esprit des gens le gagnant EST un joueur, si il y a un gagnant !
Après oui, foutrement mieux gameStatus.winner.isPlayer() que Player.isPlayer(gameStatus.winner) ! ça pour sûr !